share/locale/en.json

```diff
@@ -1,165 +1,165 @@
+    "gui_start_server": "Start OnionSharing",
+    "gui_stop_server": "Stop OnionSharing",
+    "gui_stop_server_shutdown_timeout": "Stop OnionSharing ({}s remaining)",
+    "gui_stop_server_shutdown_timeout_tooltip": "OnionShare expires at {}",
+    "gui_copy_url": "Copy OnionShare Address",
     "gui_copy_hidservauth": "Copy HidServAuth",
-    "gui_downloads": "Download History",
-    "gui_downloads_window_tooltip": "Show/hide downloads",
+    "gui_downloads": "File History",
+    "gui_downloads_window_tooltip": "Show/hide OnionSharings",
     "gui_no_downloads": "No downloads yet.",
     "gui_canceled": "Canceled",
-    "gui_copied_url_title": "Copied OnionShare address",
-    "gui_copied_url": "The OnionShare address has been copied to clipboard",
-    "gui_copied_hidservauth_title": "Copied HidServAuth",
-    "gui_copied_hidservauth": "The HidServAuth line has been copied to clipboard",
+    "gui_copied_url_title": "Copied",
+    "gui_copied_url": "OnionShare address copied to clipboard",
+    "gui_copied_hidservauth_title": "Copied",
+    "gui_copied_hidservauth": "HidServAuth line copied to clipboard",
     "gui_starting_server1": "Starting Tor onion service…",
     "gui_starting_server2": "Compressing files…",
     "gui_please_wait": "Starting… Click to cancel",
     "error_hs_dir_cannot_create": "Cannot create onion service dir {0:s}",
-    "error_hs_dir_not_writable": "onion service dir {0:s} is not writable",
+    "error_hs_dir_not_writable": "Onion service dir {0:s} is write protected",
     "using_ephemeral": "Starting ephemeral Tor onion service and awaiting publication",
-    "gui_download_progress_complete": "%p%, Time Elapsed: {0:s}",
-    "gui_download_progress_starting": "{0:s}, %p% (Computing ETA)",
+    "gui_download_progress_complete": "%p%, {0:s} elapsed.",
+    "gui_download_progress_starting": "{0:s}, %p% (calculating ETA)",
     "gui_download_progress_eta": "{0:s}, ETA: {1:s}, %p%",
     "version_string": "OnionShare {0:s} | https://onionshare.org/",
-    "gui_quit_title": "Transfer in Progress",
-    "gui_quit_warning": "You're in the process of sending files. Are you sure you want to quit OnionShare?",
+    "gui_quit_title": "Not so fast",
+    "gui_quit_warning": "Quitting now means files underway won't finish.",
     "gui_quit_warning_quit": "Quit",
     "gui_quit_warning_dont_quit": "Cancel",
-    "error_rate_limit": "An attacker might be trying to guess your address. To prevent this, OnionShare has automatically stopped the server. To share the files you must start it again and share the new address.",
-    "zip_progress_bar_format": "Compressing files: %p%",
-    "error_stealth_not_supported": "To create stealth onion services, you need at least Tor 0.2.9.1-alpha (or Tor Browser 6.5) and at least python3-stem 1.5.0.",
+    "error_rate_limit": "Too many guesses means someone could be trying to work out the address of your files. OnionSharing has stopped. Start it again and send the new address to share them.",
```

Comment:
Hmm I like this, but the user doesn't necessarily realize that someone is making "guesses" so I think it's confusing to start out saying "Too many guesses."

Maybe just: "Someone could be trying to work out your OnionShare address. OnionShare has stopped. Start it again and send the new address."

(I also reworded a bit because now OnionShare isn't just for sending someone files, it's also for receiving them)